### PR TITLE
Marcospiau readmes fix

### PR DESCRIPTION
**Pull request by marcospiau**

### CODE REVIEW

File: vocab/README.md [hunk 1]
```diff
@@ -1,38 +1,44 @@
...
 ```
-Comando: ./wikiextractor/WikiExtractor.py ptwiki-20200601-pages-articles.xml -o ptwiki-parsed-full
-Output final:
+Command: ./wikiextractor/WikiExtractor.py ptwiki-20200601-pages-articles.xml -o ptwiki-parsed-full
+
+Final output:
 INFO: 6286558   Bairro SAAL da Meia Praia
 INFO: 6286559   Capela Nossa Senhora de Montserrat
-INFO: 6286560   JoÃ£o Paulo Diniz (jornalista)
+INFO: 6286560   Joãoo Paulo Diniz (jornalista)
```
Comment (rodrigonogueira4) [inline vocab/README.md:15]: An extra "o"?

File: vocab/README.md [hunk 2]
```diff
@@ -1,38 +1,44 @@
...
+
 ```
-**Usando o tokenizer treinado**
+The files generated by this command are available in the folder `./saved_models/spm_32000_unigram` .
 
-Os arquivos com os tokenizers treinados podem ser encontrado na pasta ./saved_models. Abaixo temos exemplos de códigos de como fazer uso desse tokenizer.
+**Using the tokenizer**
 
-```
+We made this tokenizer available for use trough direct download on [🤗Transformers API](https://github.com/huggingface/transformers), so it's probably easier to use it this way. For that, one can follow the instructions on this repo's main page README (this is the tokenizer used on all models containing `"portuguese-vocab"` on its name).
```
Comment (rodrigonogueira4) [inline vocab/README.md:37]: trough -> through

Comment (marcospiau) [reply vocab/README.md]: Fixed

File: vocab/README.md (hunk 1 above)
Comment (marcospiau) [reply vocab/README.md]: Fixed

**Pull request merged**